tldr-page.go

```diff
@@ -13,6 +13,7 @@ import (
 const ( // Usage for changing the color of text
 	colorRed   = "\033[31m"
```

Comment:
If red isn't being used then we can remove it.

Reply:
Done